### PR TITLE
MI-285 Restructured LTI config structure

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Restructured LTI config structure to make querying easier and to accommodate more items such as default channel list.

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -1240,6 +1240,10 @@
     "id": "api.lti.signup.app_error.form.parse_failed",
     "translation": "There was an error with LTI signup attempt. Please try again or contact your system administrator. Error code LTI-7"
   },
+  {
+    "id": "api.lti.signup.create_user_failed",
+    "translation": "There was an error with LTI signup attempt. Please try again or contact your system administrator. Error code LTI-9"
```
Comment (chetanyakan) [inline i18n/en.json]: Remove this

File: config/config-dev.json [hunk 2]
```diff
@@ -328,55 +328,41 @@
...
-                            "NameRegEx": "^riff_Team_"
-                        }
-                    ]
-                }
+        "LMSs": {
+            "Emeritus": {
+
             },
-            {
-                "Name": "Appsembler",
```
Comment (chetanyakan) [inline config/config-dev.json]: Check if we would need a name field as well.

File: model/edx.go [hunk 3]
```diff
@@ -17,21 +17,29 @@ const (
...
+	Name        string
+	DisplayName string
 }
 
 type EdxLMS struct {
-	Name         string
-	Type         string
-	OAuth        LMSOAuthSettings
-	UserChannels EdxUserChannelsSettings
+	Name				string
```
Comment (chetanyakan) [inline model/edx.go]: Run `gofmt`.

**Review by chetanyakan — CHANGES_REQUESTED** [review]

File: model/edx.go (hunk 3 above)
Comment (harshilsharma63) [reply model/edx.go]: Done.

**Pull request merged**